migrations/20200322171627_tags.js

```diff
@@ -0,0 +1,53 @@
+exports.up = async knex => {
+  const buildTableSchema = [
+    {
+      tableName: "tag",
+      create: t => {
+        t.increments("id").primary();
+        t.text("group");
+        t.text("name").notNullable();
+        t.text("description").notNullable();
+        t.boolean("is_deleted").defaultTo(false);
+        t.timestamp("created_at")
+          .notNullable()
+          .defaultTo(knex.fn.now());
+        t.timestamp("updated_at")
+          .notNullable()
+          .defaultTo(knex.fn.now());
+        t.integer("organization_id").notNullable();
+        t.foreign("organization_id").references("organization.id");
```

Comment:
can you wrap the `.foreign` call in a `if (!isSqlite) {` (see init_db.js file for examples and what that variable is) -- annoying, but necessary